crates/networking/manager/src/service.rs

```diff
@@ -165,41 +177,91 @@ impl ManagerService {
 
         let block_range_syncer = BlockRangeSyncer::new(beacon_chain.clone(), p2p_sender.clone());
 
+        let sync_committee_subscriptions = Arc::new(RwLock::new(HashMap::new()));
+        let sync_committee_subnets = Arc::new(RwLock::new(SyncCommitteeSubnets::new()));
+
         Ok(Self {
             beacon_chain,
             manager_receiver,
             p2p_sender: P2PSender(p2p_sender),
             network_state,
             block_range_syncer,
             ream_db,
+            sync_committee_subscriptions,
+            sync_committee_subnets,
         })
     }
 
+    /// Fetch the latest BeaconState from the DB by highest slot.
+    fn get_latest_beacon_state(db: &ReamDB) -> anyhow::Result<Option<BeaconState>> {
+        let highest_slot = db
+            .slot_index_provider()
+            .get_highest_slot()?
+            .ok_or_else(|| anyhow::anyhow!("No highest slot found in database"))?;
```

Comment:
```suggestion
            .ok_or_else(|| anyhow!("No highest slot found in database"))?;
```

please remove this path qualifer from the grest of the pr